src/components/common/YourCompany/StartDate.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <section id="business-start-date">
+    <v-row no-gutters>
+      <v-col cols="3" class="pr-2">
+        <label :class="{'error-text': invalidSection}">
+          <strong>Business Start Date</strong>
+        </label>
+        <v-chip
+          v-if="isCorrected"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          <span>{{ editedLabel }}</span>
+        </v-chip>
+      </v-col>
+
+      <!-- Info tooltip-->
+      <v-col cols="9">
+        <span v-if="!onEditMode" class="info-text mr-1">{{ recognitionDateTime }}</span>
+        <v-tooltip v-if="!isCorrectionFiling"
+                    top
+                    content-class="top-tooltip"
+                    transition="fade-transition"
+                    nudge-right="3"
+        >
+          <template v-slot:activator="{ on }">
+            <v-icon v-on="on" class="info-icon">mdi-information-outline</v-icon>
+          </template>
+          <span>If the business start date is incorrect, it must be corrected through a correction filing.</span>
+        </v-tooltip>
+
+        <!-- Edit mode -->
+        <div v-if="onEditMode" cols="12" sm="3" class="pr-4">
+          <label class="start-date-title title-label">Start Date</label>
+          <DatePickerShared
+            title="Start Date"
+            nudge-right="80"
+            nudge-top="15"
+            :minDate="startDateMinStr"
+            :maxDate="startDateMaxStr"
+            @emitDate="onOkClicked($event)"
+          />
+        </div>
+        <div v-if="onEditMode" class="float-right mb-2">
```

Comment:
Since you have a couple of sequential divs with the same v-if, and to make it easier to fold code, I recomment a template component around these divs and move the v-if to it.

Reply:
Okay will do!